styles.css

```diff
@@ -0,0 +1,31 @@
+.main-doc{
+
+}
+body{
+ background-color: aqua;
+ font-family: Arial, Helvetica, sans-serif ;
+ margin: 0;
```

Comment:
Please always try to to explain why you request a change - it might be obvious for you but not for someone else.

you can say like, 'I suggest to use the example fonts and colors as it will help us follow the design properly'

Reply:
Please try to follow the design fonts and color properties as requested in the exercise.<https://technical-documentation-page.freecodecamp.rocks/

Reply:
As requested by the previous reviewer please let's always try to explain why we request a change - it might be obvious for you but not for someone else.

**[OPTIONAL]** You can create an inline link by wrapping link text in brackets [ ] and then wrapping the URL in parentheses ( ). to create sophisticated formatting for your prose and code on GitHub. 

e.g => `[Click here](https://link_to_the_website)`

You can learn more [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)